react-native-git-upgrade/cli.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+require('babel-register')({
+  presets: [
+    require('babel-preset-es2015-node'),
+    require('babel-preset-stage-3')
+  ],
+  only: /(react-native-git-upgrade\/(?!(node_modules)))|(local-cli\/generator)/,
```

Comment:
Good point about the trailing commas! For me local-cli/generator ran fine with node 6.9 and 7,  I never know which features are standard in which version of JS :)

> I think we should be consistent: if the generator is transpiled with react-native upgrade, it must be the same with react-native-git-upgrade.

I agree.